### PR TITLE
[WIP]: Documentation/traceability --> ./usage-information

### PR DESCRIPTION
**Pull request by davidkartuzinski**
> ⚠️ We have stopped merging PRs for now to the Strapi core.<br><br>
> The reason is that we are developing new architecture for the admin panel and for the plugins.<br>
> This new architecture will provide stability of the Strapi core as we approach the release of Beta.<br>
> We appreciate and welcome all your contributions, but until further notice, please do not submit a PR as it will not be merged.<br>
> Furthermore, you will have to rewrite it based on the new architecture.

[WIP]: Documentation/traceability --> ./usage-information

I propose the following for the new wording of this page.  Along with additional clarifying comments within the text, typos have been fixed.  I have rearranged the order of the paragraphs to better suit the message as intended.

#### Description:

Strapi has a feature to measure usage.  This feature collects usage information.  This document which describes exactly what is being collected is now revised to clarify several points in the text and to rearrange the paragraph to improve readability.

I didn't change the file name, so as to preserve the comparison.  But this should change to whatever is decided upon.

#### My PR is a:
- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix #issueNumber
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:
- [ ] Admin
- [x] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:
- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
LGTM

File: docs/3.0.0-beta.x/advanced/traceability.md [hunk 1]
```diff
@@ -1,44 +1,53 @@
...
+  "strapi": {
+    "uuid": "7b581c0d-89b7-479e-b379-a76ab90b8754"
+  },
+  "license": "MIT"
+}
+```
+
+### Questions?
+
+Should you have any questions regarding this feature, feel free to email [hi@strapi.io](mailto:hi@strapi.io), and we will answer you right away.
```
Comment (derrickmehaffy) [inline docs/3.0.0-beta.x/advanced/traceability.md]: `and we will answer you as soon as possible`

Not a good idea to make "promises" that might be difficult to keep.

Comment (davidkartuzinski) [reply docs/3.0.0-beta.x/advanced/traceability.md]: Agreed. @derrickmehaffy

**Comment by Aurelsicoko** [comment]
@davidkartuzinski LGTM! Thank you for reviewing this PR, feel free to rename the file 👍

**Pull request merged**